bucket/tfsec.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "0.58.9",
+    "description": "tfsec uses static analysis of your terraform templates to spot potential security issues.",
+    "homepage": "https://github.com/aquasecurity/tfsec",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/aquasecurity/tfsec/releases/download/v0.58.9/tfsec-windows-amd64.exe",
```

The executable needs re-mapping.

```suggestion
            "url": "https://github.com/aquasecurity/tfsec/releases/download/v0.58.9/tfsec-windows-amd64.exe#/tfsec.exe",
```